website/content/en/blog/2025/query-rejection.md

```diff
@@ -103,9 +103,26 @@ Because this request matches all the configured attributes, it will be blocked.
 
 - **Communicate with tenants.** Let affected tenants know if their queries are being blocked, and help them adjust their dashboards accordingly.
 
-## Conclusion
+## Ruler Queries
+
+Query rejection only applies to API queries and does not apply to ruler queries. However, Ruler queries are typically instant and lightweight, so a complex query‑rejection mechanism isn’t required for them. 
```

Comment:
I think a big reason here is that rules are scoped by rule group by nature. And it is hard to disable only a single rule in the rule group as other rules can depend on the disabled rule. So support rule group level rejection is sufficient for now.